tests/appverifier_tests/README.md

```diff
@@ -0,0 +1,29 @@
+# Thread_local Tests Suite
+
+This test suite is used to detect whether AppVerifier will report a memory leak caused by thread_local.
```

Comment:
This file should contain common description, details about `thread_local` should be described further

Reply:
thanks, updated